packages/assets-controllers/src/DeFiPositionsController/fetch-positions.ts

```diff
@@ -0,0 +1,77 @@
+export type DefiPositionResponse = AdapterResponse<{
```

Comment:
(personal pref) - does this DEFI api have an openAPI spec?
If yes, then we can generate these types and makes it easier to keep in sync.

Reply:
Notifications does this here:
https://github.com/MetaMask/core/blob/6eeac725430ed84f70c37283a6683e9074db56f3/packages/notification-services-controller/src/NotificationServicesController/types/on-chain-notification/schema.ts#L3-L12

Reply:
We will need to look into this, as it'd definitely be better to automatically generate the types.